<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationCheckDialog</class>
 <widget class="QWidget" name="ConfigurationCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration Load Status</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>STATUS DESCRIPTION HERE</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="detailsFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Errors found in configuration :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigurationCheckDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
